tests/flow/random_graph.py

```
@@ -0,0 +1,262 @@
+                           "R": {"source": "N", "target": "M", "count": 500}})
+    """
+    nodes = []
+    edges = []
+
+    labels_count = randint(1, 10)
+    for i in range(0, labels_count):
+        count = randint(10, 100)
+        properties = randint(0, 10)
+        labels = [f"N{i}"]
+        for j in range(0, randint(0, 5)):
+            l = randint(1, labels_count)
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)